modules/data-table/main/index.coffee

```diff
@@ -27,6 +34,13 @@ columnOptionLookup = (options, name, id) ->
   else
     options[name]
 
+toCriteriaItems = (list) ->
+  new hx.Set(list).values().map (item) ->
```

Comment:
Wasn't aware of `hx.unique` - Missed it because it's so close to the bottom of the [Util](https://www.hexagonjs.io/docs/util/) api section...